test/trento_web/auth/access_token_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule TrentoWeb.AccessTokenTest do
+  @moduledoc false
+
+  alias TrentoWeb.Auth.AccessToken
+
+  use ExUnit.Case
+
+  import Mox
+
+  setup [:set_mox_from_context, :verify_on_exit!]
+
+  setup do
+    expect(
+      Joken.CurrentTime.Mock,
+      :current_time,
+      3,
+      fn ->
+        1_671_641_814
```

Comment:
this number can go to a module attribute for clarity since we repeat it a lot in the tests below